app/src/main/java/org/stepik/android/view/course_list/ui/adapter/delegate/CourseListItemAdapterDelegate.kt

```diff
@@ -30,9 +30,9 @@ class CourseListItemAdapterDelegate(
     private val analytic: Analytic,
     private val onItemClicked: (CourseListItem.Data) -> Unit,
     private val onContinueCourseClicked: (CourseListItem.Data) -> Unit,
-    private val isHandleInAppPurchase: Boolean,
     private val defaultPromoCodeMapper: DefaultPromoCodeMapper,
     private val displayPriceMapper: DisplayPriceMapper,
+    private val isIAPFlowEnabled: Boolean,
```

Comment:
Do we really need to have `isIAPFlowEnabled` here? Can we look just on `NotEnrolledMobileTier`?